src/applications/login/components/TransitionAccountPage.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { transitionMHVAccount } from 'platform/user/authentication/selectors';
+import TransitionAccountCTA from './TransitionAccountCTA';
+import IdentityWizard from './IdentityWizard';
+
+export default function TransitionAccount() {
+  // TODO: wait for BE team to add to User model
+  const canTransition = useSelector(transitionMHVAccount);
```

Comment:
Once both of our PRs are in, and the backend has done the implementation, we should sync and make sure we utilize the same logic for this determination on both the modal and the page.